docs/internals/contributing/bugs-and-features.txt

```diff
@@ -29,6 +29,11 @@ general points:
   likely to get lost. If a particular ticket is controversial, please move the
   discussion to |django-developers|.
 
+.. Note::
+
+    Django does not use GitHub for bug reports or feature requests.
```

Comment:
Well, someone who assumes we use GitHub issues might never find the new-ticket page in the first place. I don't know how likely it is that such a person will read the contributing guide first...

IMO the second sentence here ("All bug reports or feature requests...") is good, but the callout note and the phrasing "Django does not use GitHub..." is probably overkill, and possibly confusing.